src/components/search/GlobalSearch.tsx

```diff
@@ -17,25 +17,17 @@ export default function GlobalSearch() {
   const [loading, setLoading] = useState(false)
   const inputRef = useRef<null | HTMLInputElement>(null)
 
-  useEffect(() => {
-    inputRef?.current?.focus()
-  }, [searchOpen])
-
   const openSearch = () => {
     setSearchOpen(true)
-    document.body.style.overflow = 'hidden'
     inputRef?.current?.focus()
   }
 
-  const reset = () => {
+  const closeSearch = () => {
```

Comment:
I think `reset` was a better name, or `onClose`.

This is an event on close, so technically saying `closeSearch` is inaccurate as this code isn't closing the search.

Reply:
Makes sense. Renamed to `onClose` in latest commit!